homeassistant/components/arve/sensor.py

```diff
@@ -0,0 +1,117 @@
+"""Support for Arve devices."""
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from asyncarve import ArveSensProData
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    CONCENTRATION_PARTS_PER_MILLION,
+    PERCENTAGE,
+    UnitOfTemperature,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import ArveCoordinator
+from .entity import ArveDeviceEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class ArveDeviceEntityDescription(SensorEntityDescription):
+    """Describes Arve device entity."""
+
+    value_fn: Callable[[ArveSensProData, str], float | int]
```

Comment:
```suggestion
    value_fn: Callable[[ArveSensProData, str], float | int] = lambda arve_data, key: getattr(arve_data, key.lower())
```
and then you can remove all the other references below